src/v/redpanda/admin_server.cc

```diff
@@ -1893,17 +1917,20 @@ void admin_server::register_hbadger_routes() {
      */
 
     if constexpr (!finjector::honey_badger::is_enabled()) {
-        ss::httpd::hbadger_json::get_failure_probes.set(
-          _server._routes, [](std::unique_ptr<ss::httpd::request>) {
+        register_route<superuser>(
+          ss::httpd::hbadger_json::get_failure_probes,
+          [](std::unique_ptr<ss::httpd::request>) {
               ss::httpd::hbadger_json::failure_injector_status status;
               status.enabled = false;
               return ss::make_ready_future<ss::json::json_return_type>(
                 std::move(status));
           });
         return;
     }
-    ss::httpd::hbadger_json::get_failure_probes.set(
-      _server._routes, [](std::unique_ptr<ss::httpd::request>) {
+
+    register_route<user>(
+      ss::httpd::hbadger_json::get_failure_probes,
+      [](std::unique_ptr<ss::httpd::request>) {
```

Comment:
it looks like the exact same route is registered right above this with superuser auth required?

```
        register_route<superuser>(
           ss::httpd::hbadger_json::get_failure_probes,
           [](std::unique_ptr<ss::httpd::request>) {
```

Reply:
Yep, you're right.  First one should have been `user` too.